### DISCUSSION TITLE
/how-to-setup-invoiceninja-v4-with-traefik-v2-nginx-with-tls/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#136] [Announcements]
How to setup InvoiceNinja v4 with Traefik v2 + nginx with TLS
        
[https://blog.ktz.me/how-to-setup-invoiceninja-v4-with-traefik-v2-nginx-with-tls/](https://blog.ktz.me/how-to-setup-invoiceninja-v4-with-traefik-v2-nginx-with-tls/)

**Comment by migratecomments** [comment]
```header
Originally posted by Nils Kretschmer on 2021-01-12 00:08
```

Thank you for this nice Tutorial. I have a question regarding mounting the config files into the containers. Do I need to add additional volumes to those which are already configured in your docker-compose file? Or is it sufficent to just create the directories for the existing volumes and add those config files before spinning up?

**Comment by migratecomments** [comment]
```header
Originally posted by Dwijadas on 2021-08-02 20:21
```

Cant get it working ... something wrong at NGIX config

**Reply by ThomasSonck** [reply]
+1 assets are not loading here
